modules/serverless-autoscaling-dashboard-scrape-time.adoc

```diff
@@ -0,0 +1,6 @@
+[id="serverless-autoscaling-dashboard-scrape-time_{context}"]
+= Scrape time
+
+The *Scrape Time* graph shows the timeline of scrape times for each revision. Since autoscaling makes scaling decisions based on the metrics coming from service pods, high scrape times might cause delays in autoscaling when workload changes.
```

Comment:
```suggestion
The *Scrape Time* graph displays the timeline of scrape times for each revision. Since autoscaling makes scaling decisions based on the metrics coming from service pods, high scrape times might cause delays in autoscaling when workload changes.
```